roles/kubespray_defaults/vars/main/main.yml

```diff
@@ -7,14 +7,14 @@ kube_next: "{{ ((kube_version | split('.'))[1] | int) + 1 }}"
 kube_major_next_version: "1.{{ kube_next }}"
 
 pod_infra_supported_versions:
+  '1.34': '3.10'
   '1.33': '3.10'
   '1.32': '3.10'
-  '1.31': '3.10'
 
 etcd_supported_versions:
+  '1.34': "{{ (etcd_binary_checksums['amd64'].keys() | select('version', '3.7', '<'))[0] }}"
```

Comment:
3.7? 3.6?

Reply:
The etcd 3.6 is supported by kubernetes 1.34, so I set this to <3.7.
By this PR, no etcd 3.6 hashes exist in the checksums.yml, so latest etcd 3.5.x will be selected.

Shall I keep <3.6 for this PR? (and changed to <3.7 by other PR?)

Reply:
Ok. I will keep it as is 3.6.